migrations/versions/480_add_briefs_table.py

```diff
@@ -0,0 +1,36 @@
+"""Add briefs table
+
+Revision ID: 480
+Revises: 470
+Create Date: 2016-01-25 15:45:50.363821
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '480'
+down_revision = '470'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
```

Comment:
I think it's fine either way. It might be surprising during review, but once it's merged there's no expectation that the initial table creation migration describes the current table schema, so I don't see a problem with leaving this as is.